package/yast2-services-manager.spec

```diff
@@ -35,6 +35,8 @@ Requires:       ruby
 # Firewalld interfaces helpers
 Requires:       yast2 >= 4.0.51
 Requires:       yast2-ruby-bindings >= 1.2.0
+# To show service logs
+Suggests:	yast2-journal >= 4.1.1
```

Comment:
why suggest and not Recommends?

Reply:
Recommends will try to install it. I would say that this package should no try to install the journal.